projects/bp-gallery/src/components/views/exhibitions/ExhibitonUtils.tsx

```diff
@@ -827,29 +832,8 @@ export const ExhibitionStateManager = ({
   useEffect(() => {
     setIdealot(buildDragElements(exhibition.idealot_pictures));
   }, [setIdealot, exhibition]);
-  const [updateExhibitionPicture] = useUpdateExhibitionPictureMutation();
 
-  const [updateExhibitionSection] = useUpdateExhibitionSectionMutation();
-
-  const [updateExhibitionSource] = useUpdateExhibitionSourceMutation();
-
-  const [updateExhibition] = useUpdateExhibitionMutation();
-
-  const [createSource] = useCreateExhibitionSourceMutation();
-
-  const [createSection] = useCreateExhibitionSectionMutation();
-
-  const [createExhibitionPicture] = useCreateExhibitionPictureMutation();
-
-  const databaseSaver = ExhibitionDatabaseSaver(
-    updateExhibitionSection,
-    updateExhibitionPicture,
-    updateExhibitionSource,
-    updateExhibition,
-    createSection,
-    createSource,
-    createExhibitionPicture
-  );
+  const databaseSaver = ExhibitionDatabaseSaver();
```

Comment:
At this point the ExhibitionDatabaseSaver is basically just a hook, maybe rename it to useExhibitionDatabaseSaver to better represent that? Also, it currently gets exported but only used in the ExhibitionUtils. You should probably either remove the export or move it to its own helper hook file.